website/docs/d/lambda_layer_version.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "aws"
+page_title: "AWS: aws_lambda_layer_version"
+sidebar_current: "docs-aws-datasource-lambda-layer-version"
```

Comment:
Missing easy-to-forget sidebar link in `website/aws.erb` 😅

Reply:
whoops yeah, sorry about that 😬